### PR TITLE
[LIB-381][LIB-324] Add ability to login with LinkedIn as a User,Add ability to login with Twitter for users

### PR DESCRIPTION
**Pull request by trapi**

### CODE REVIEW

File: SyncanoIntegrationTests/UserLoginSpec.m [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ *  You should set GOOGLE_TOKEN_KEY, FACEBOOK_TOKEN_KEY, TWITTER_TOKEN_KEY, LINKEDIN_TOKEN_KEY for this tests to work
+ */
+SPEC_BEGIN(UserLoginSpec)
+
+describe(@"User login", ^{
+    
+    NSDictionary *environment = [[NSProcessInfo processInfo] environment];
+    NSString *apikey = environment[@"API_KEY"];
```
Comment (lifcio) [inline SyncanoIntegrationTests/UserLoginSpec.m]: camelCase

**Comment by lifcio** [comment]
lgtm

File: SyncanoIntegrationTests/UserLoginSpec.m (hunk 1 above)
Comment (trapi) [inline SyncanoIntegrationTests/UserLoginSpec.m]: Right.. But actually I copy & paste it from an existing test.. ;)

Comment (lifcio) [inline SyncanoIntegrationTests/UserLoginSpec.m]: I noticed, but I could comment on your code only ;) We will have to update it everywhere over time.

**Pull request merged**